src/web/app/pages-help/instructor-help-page/instructor-help-panel/instructor-help-panel.component.scss

```diff
@@ -1,7 +1,27 @@
 .card-header {
   cursor: pointer;
+
+  .fa-anchor {
+    display: none;
+    opacity: 30%;
+  }
+
+  &:hover {
```

Comment:
A minor note: While we don't officially support mobile devices, reveal on hover is generally bad practice for mobile accessibility (for obvious reasons). In addition, I suspect this might lead to "hover traps" on some mobile devices, where 
Case 1) The user tries to expand the panel but instead clicks on the invisible anchor link icon.
Case 2) The user tries to expand the panel, but clicking on the panel activates hover effect on the panel instead, which reveals the icon instead of expanding the panel.

If it's not too much effort, we might want to consider adding different styles for mobile screens (making the icon visible or disabling this feature entirely). That being said, not a big issue since we don't officially support mobile,  can proceed without too.

Reply:
@samuelfangjw Disabled this feature for screens smaller than 1000px.